content/en/news/support/announcing-1.24-eol/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.24 ends on June 19, 2025
+subtitle: Support Announcement
+description: Upcoming Istio 1.24 end of life announcement.
+publishdate: 2025-06-11
```

Comment:
```suggestion
publishdate: 2025-05-20
```
This should have been published on May 20th, what do we want to set for `publishdate` here?

Reply:
```suggestion
publishdate: 2025-06-12
```
Due to timezones this notice won't merge until at least tomorrow.